lib/rules/use-colors/__tests__/index.test.js

```diff
@@ -0,0 +1,95 @@
+/*
+ Copyright 2020 Skyscanner
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+
+ http://www.apache.org/licenses/LICENSE-2.0
+
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+ */
+
+const { lint } = require('stylelint');
+
+const config = {
+  plugins: ['./index.js'],
+  rules: {
+    'backpack/use-colors': [true],
+  },
+};
+
+it('passes when setting color', async () => {
+  const {
+    results: [{ warnings }],
+  } = await lint({
+    code: 'a { color: $bpk-color-sky-blue; font-size: 14rem }',
+    config,
+  });
```

Comment:
async await, nice!

Reply:
Can't take all the credit this was stylelints recommendation 😄